packages/shop-mobile-expo/package.json

```diff
@@ -0,0 +1,105 @@
+{
+	"name": "@ever-platform/shop-mobile-expo",
+	"description": "Ever Shop Mobile App build with expo",
+	"version": "1.0.0",
```

Comment:
@Neosoulink Let's put here same version as other apps in our platform for consistency

Reply:
@Neosoulink I did it together with other changes, make sure you pull latest!

Reply:
Yes sure